modules/light-clients/07-tendermint/module.go

```diff
@@ -1,6 +1,53 @@
 package tendermint
 
-// Name returns the IBC client name
-func Name() string {
+import (
+	"encoding/json"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/codec"
+	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	"github.com/grpc-ecosystem/grpc-gateway/runtime"
+	"github.com/spf13/cobra"
+)
+
+var _ module.AppModuleBasic = AppModuleBasic{}
+
+// AppModuleBasic defines the basic application module used by the tendermint light client.
+type AppModuleBasic struct{}
+
+// Name returns the tendermint module name.
+func (AppModuleBasic) Name() string {
 	return SubModuleName
```

Comment:
Not related to this PR, so could be done in a separate one, but I was just thinking if we should rename this `SubModuleName` to `ModuleName`? And `SubModuleName` is currently defined [errors.go](https://github.com/cosmos/ibc-go/blob/main/modules/light-clients/07-tendermint/errors.go#L8), but maybe could also moved to a new keys.go file like we have in other modules.

Same thing would apply to the solomachine light client, I guess.

Reply:
Yes agreed. Can you open an issue?

Reply:
Cool. I opened https://github.com/cosmos/ibc-go/issues/2833 and https://github.com/cosmos/ibc-go/issues/2834.